### DISCUSSION TITLE
Embed configurator in btcpay

### DISCUSSION THREAD

**Discussion by Kukks** [btcpayserver/btcpayserver#2720] [Plugins & integrations]
Find a way to run the btcpay configurator inside the btcpay process. I've tried a few ways but nothing worked in a good way.

**Comment by Eskyee** [comment]
I love the idea of this, @Kukks . 
maybe this could help migrate our current users from btcpayjungle, or third party hosts to their own full node.
another idea ?? could there be a way of migrating the current user data, to their new full bitcoin node. ??
while setting up the configurator. ?

**Comment by pavlenex** [comment]
@Kukks plugin?

**Comment by Kukks** [comment]
Yes

**Comment by pavlenex** [comment]
Closing this since it's a plugin idea and adding to discussions.
